docs/api.rst

```diff
@@ -211,18 +211,72 @@ API Methods
         }
 
 
-   **Description of the JSON response attributes**:
+    **Description of the JSON response attributes**:
 
-   The response contains mappings between the metric names and an array of
-   their timestamp-value pairs, where the pairs in the array are in ascending
-   order of their timestamp values (from the earliest time to the latest time).
-   
-   Each pair contains the timestamp under the ``x`` field, and is formatted as
-   the number of milliseconds elapsed since 1 January 1970 00:00:00 UTC.
+    The response contains mappings between the metric names and an array of
+    their timestamp-value pairs, where the pairs in the array are in ascending
+    order of their timestamp values (from the earliest time to the latest time).
 
-   Each pair contains the value under the ``y`` field, and is formatted as a
-   json number.
+    Each pair contains the timestamp under the ``x`` field, and is formatted as
+    the number of milliseconds elapsed since 1 January 1970 00:00:00 UTC.
 
+    Each pair contains the value under the ``y`` field, and is formatted as a
+    json number.
```

Comment:
I'm not sure why this comes up so strangely in the diff. I just tabbed this paragraph to the right. The original indentation was 2 spaces instead of 4, so I corrected that.